data/experiment-recipe-samples/rollout-desktop.json

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
We should include a desktop version number in this filename, I think it's 95 but maybe confirm with @piatra

Reply:
95 if this lands before next Tuesday Dec 7th but I think less important because Experimenter can make changes outside of the train model. What people are interested in is when both Experimenter and Firefox support X feature so the answer most likely will come from the Firefox side.

Reply:
In this case we'll land the Firefox code in 97 so in practice we won't be able to launch/test except on >= 97.